app/helpers/chat_migration_message.rb

```diff
@@ -0,0 +1,24 @@
+module ChatMigrationMessage
+
+  DISPLAY_MESSAGE_YEAR  = 2016
+  DISPLAY_MESSAGE_MONTH = 2
+  DISPLAY_MESSAGE_DAY   = 6
+
+  class Message
+    def initialize
+      @migration_date = Date.new(DISPLAY_MESSAGE_YEAR, DISPLAY_MESSAGE_MONTH, DISPLAY_MESSAGE_DAY)
+    end
+
+    def display_message?
+      Date.today == @migration_date
+    end
+  end
+
+  def display_chat_migration_message?
+    ChatMigrationMessage::Message.new.display_message?
+  end
+
+  def alternate_hours_active?
+    display_chat_migration_message?
+  end
+end
```

Comment:
Hi @JiggyPete I've tried to change this to the above and the tests began to fail, this is more than likely due to my lack of back end skills, any ideas?